interfaces/builtin/u2f_devices.go

```diff
@@ -121,6 +121,11 @@ var u2fDevices = []u2fDevice{
 		VendorIDPattern:  "0483",
 		ProductIDPattern: "a2ac",
 	},
+	{
+		Name:             "Ledger Blue + Nano S + Nano X",
+		VendorIDPattern:  "2c97",
+		ProductIDPattern: "0000|0001|0004",
```

Comment:
https://github.com/Yubico/libu2f-host/blob/master/70-u2f.rules#L68-L69 doesn't list the Ledger Blue (0000), but http://www.linux-usb.org/usb.ids does. Googling, it seems the blue is a u2f device. Can someone confirm? Once confirmed, +1 to apply.

Reply:
It seems to be https://github.com/LedgerHQ/app-u2f 

For the record: I also noticed that the referenced document doesn't list the blue product number but it looked sensible so I didn't ask about it.

Reply:
Ahem, I was not up to date with our latest firmware upgrade:
We used to have these values but we recently changed the way our devices announce their productID: https://www.ledger.com/windows-10-update-sunsetting-u2f-tunnel-transport-for-ledger-devices/
So I have to make a PR first on https://github.com/Yubico/libu2f-host (I will also add the Blue product there).
Meanwhile I'm updating the productIDs here with their new values. Sorry for the back-pedaling.

Reply:
PR waiting here: https://github.com/Yubico/libu2f-host/pull/151

Reply:
Please clarify the product IDs as asked in the referenced PR. Right now it seems that this is wrong in one or other way.

Reply:
https://github.com/Yubico/libu2f-host/pull/151 has been updated and merged.
I just updated this one to match what I've done there (adding back old productIDs next to updated ones)